Magento_Customer/templates/form/forgotpassword.phtml

```diff
@@ -1,33 +1,33 @@
 <?php
 /** @var $block \Magento\Customer\Block\Account\Forgotpassword */
 ?>
-<form class="form forgot-password"
-      action="<?= /* @escapeNotVerified */ $block->getUrl('*/*/forgotpasswordpost') ?>"
-      method="post"
-      id="form-validate"
-      data-mage-init='{"validation":{}}'
->
-    <p class="paragraph">
+<div class="forgot-password">
+    <p>
         <?= /* @escapeNotVerified */ __('Please enter your email address below to receive a password reset link.'); ?>
     </p>
```

Comment:
I have some proposition to make this code even more accessible:  you can add to this `<p>` an `id` attribute and use it as a value for an attribute `aria-labelledby` in `<form>`, you'll have additional tips for screen reader. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

Reply:
Great, thanks!